prosody/rootfs/defaults/conf.d/jitsi-meet.cfg.lua

```diff
@@ -22,6 +22,22 @@ http_default_host = "{{ .Env.XMPP_DOMAIN }}"
 
 {{ $ENABLE_XMPP_WEBSOCKET := .Env.ENABLE_XMPP_WEBSOCKET | default "1" | toBool }}
 {{ $PUBLIC_URL := .Env.PUBLIC_URL | default "https://localhost:8443" -}}
+{{ $TURN_PORT := .Env.TURN_PORT | default "443" }}
+
+{{ if .Env.TURN_CREDENTIALS }}
+turncredentials_secret = "{{.Env.TURN_CREDENTIALS}}";
+{{ end }}
+
+{{ if .Env.TURN_HOST }}
+
+turncredentials_host = "{{ .Env.TURN_HOST }}";
+turncredentials_port = "{{ $TURN_PORT }}";
+
+turncredentials = {
+  { type = "turn", host = "{{ .Env.TURN_HOST }}", port = "{{ $TURN_PORT }}", transport = "tcp" },
+  { type = "turns", host = "{{ .Env.TURN_HOST }}", port = "{{ $TURN_PORT }}", transport = "tcp" }
```

Comment:
I think we should have a separate env bar for the TURNS port.

Reply:
I think those usually are at the same port. Futhermore there is only one turncredentials_port option - i don't know if that would work with two ports.

Reply:
By using the external_services module it was possible to set separate turn/turns servers